go.py

```diff
@@ -762,7 +762,7 @@ def _import(self, fn):
         self.save()
 
 
-class Root:
+class Root(object):
```

Comment:
I suggest just removing (object) so we're forcing python3. It's not needed any longer in python3 and supporting v2 will become increasingly difficult. Was this done to get it running in 2x?

Reply:
I forget what I was doing here. Didn't run it in Python 2 after I converted to Python 3. 

Fun fact, using CherryPy>=18.0.0 is dependent on running Python 3 (part of the reasons why unit tests were failing).